test/ng/compileSpec.js

```diff
@@ -4193,7 +4194,7 @@ describe('$compile', function() {
       dealoc(element);
       compile('<div><span my-component>');
       expect(componentScope).not.toBe(firstComponentScope);
-      expect(componentScope.$$isolateBindings).toBe(first$$isolateBindings);
+      expect(componentScope.$$isolateBindings).toEqual(first$$isolateBindings);
```

Comment:
Right !

In that case, it might be useful to have a test that verifies that `bindingCache` works (i.e. returns the same object for the same scope binding definition (when `attrName` is defined)).
I'm not sure though, since it might fall under the "implementation detail" category and shouldn't be tested for explicitly.

Leaving it up to you :smiley: